src/Symfony/Component/DependencyInjection/Tests/Fixtures/GetterOverriding.php

```diff
@@ -21,8 +21,11 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class GetterOverriding
+abstract class GetterOverriding
```

Comment:
Maybe should you add another test instead of editing this one to be sure that both cases are covered?